frank/geometry.py

```diff
@@ -22,12 +22,10 @@
    NOTE: The sign convention used here is xx.
 """
 
-import numpy as np
```

Comment:
In general, I see that it was adopted a convention to import `frank` modules first and external modules (e.g., `numpy`, `scipy`, ...) later.
Actually, the [correct PEP8 practice](https://www.python.org/dev/peps/pep-0008/#imports) is the other way around: 
```
    1. Standard library imports.
    2. Related third party imports.
    3. Local application/library specific imports.
```
One has to ensure frank is compliant with the distribution, not the other way around.
Hopefully `np`, or `least_squares` are not objects in `frank`, but if it was the case, `import numpy as np` as it is right now would shadow our `np` object.
So, nothing to change for now as long as it works as intended.
In the future, we might just want to adopt the right convention in the whole codebase.

Reply:
Good to note, thanks!